README.md

```diff
@@ -225,6 +225,9 @@ it is recommended not to preinstall containerd in MachineSet or BYOH Windows ins
 Windows Nodes do not support workloads created via DeploymentConfigs. Please use a normal Deployment, or other method to
 deploy workloads.
 
+### Cluster-wide proxy 
+WMCO does not support adding Windows workloads using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html) config for the OpenShift Container Platform. WMCO will not be able to automatically route proxy connections outside the cluster for the workloads added using the proxy setup.
```

Comment:
I'm having an issue understanding what `workloads added using the proxy setup` actually entails. This is a cluster wide proxy so wouldn't all workloads will be using the proxy?

Shortening it to `WMCO will not be able to automatically route proxy connections for Windows workloads` would make more sense to me.